### PR TITLE
First section of the Story

### PR DESCRIPTION
**Pull request by Jax-Tsukuyomu**
Descriptions of the main character and a few subcharacters all the way up
to the house. Welcome everyone, I plead with you to please read this and give feedback on how to make it better, This is the beginning of the story, and also if we don't like the title that can always be changed. Thanks and now if it is approved it get passed on the the second writer, first puzzle maker, media guy, and first stylist. Good Luck!

### CODE REVIEW

**Comment by dwds** [comment]
It needs to marked up in HTML, yo.

**Comment by dwds** [comment]
Also, we don't want no .docx file up in here.

**Comment by jf-42** [comment]
I italicized the few small ideas I had and some minor corrections I noticed.

In my town there is not much to hold many people's attention. The *community* college, the insignificant downtown, the *farmland*, and the forest all come together each like a stitch in a Christmas quilt. The town as a whole mostly holds a special place in the *townsfolk’s* hearts. The older generation has lived *here* since their *parent’s* generation and thereon, so everyone looks out for one another. Though this is good for a starting detective such as myself, it also makes my job just a little bit monotonous. 
	Welcome to Janesborough, home of the America's largest usable ruler. Exciting, right? The amount of tourism generated by “The Ruler” is a whopping hundred people a year. So that should paint   the current predicament that I operate in. Me? My name is Vincent Kirk. I have been promoted to detective bout two weeks ago, which has left me thoroughly bored and empty. Not because I miss the *beat* cop days, but because of the lack of cases. *Or the lack of quality cases, anyway.* Recently, I have had a case about a missing cat, a mysterious haunted bathroom in the *town’s* park, and vandalism on a few businesses. I have so much time that I can sit back in my new 'corner' office and daydream the day away, but it also gives me time to write this… 
	“Duche, duche, duche.” Someone knocks on Vincent's door. Vincent breaks from his work-deprived daze and shakes himself awake. “Yes, come on in.” Large, but particularly unimposing, the chief of the Janesborough *Police Department* walks into Vincent's room. 
	“Vincent, you're *gunna* love this! We finally got a call, an actual case. So you can stop twiddling with your thumbs and writing a Senior Thesis.” joked the chief. His large chin bounced and bulged as he laughed at his own joke. “Chief, I appreciate the sentiment, but this will probably be about another lost animal again. Oh, oh, oh! Is it a dog this time,” replied Vincent sarcastically. “No, smart-ass. Just get on phone line one” started the chief, “I left this one juuuuuust for you.” Then the chief shuffled his bulging body out the door, turning sideways to get *through* the door. 
	Vincent sat *behind* his dark-stained desk, bewildered how such a useless man became chief. As Vincent began to drift once again into his own consciousness, the phone on his desk rang. Though not particularly happy with the situation, he had been looking for more cases, no matter the subject matter. Sighing, Vincent swiftly grabs the phone and *places* it upon his ear. “Thank you for holding, this is Detective Vincent. What can I do for…” “You guys need to get down here now! Something weird is going on in the house across the street” yelled the man on the phone. “Hugh… Sir, what exactly is 'weird' about the house across the street?” questioned Vincent. 
	The man then proceeded to explain various details of the home over the *phone*. The mass of the phone call turned from actual details to incessant jumbled words ending with evil. Moreover the man on the other end of the call was what many in the town called, an old coot. Mr. Jonson, the oldest man of the town, lived in the same house for 80 years with no remaining family in the area. So his days consisted of arguing with his neighbors about Jimmy Carter, and religious ideologies. The call continued for what seem like hours to Vincent. Mentally checked out, Vincent answered all of the questions and concerns of Mr. Jonson with a swift “Mhm” or “Okay”. But as the call seemed to be coming to an end, Mr. Jonson broke down into a quiet sob. 
“*They’re* coming for me next. The aliens… they know I've contacted you. I'm sure that *they’re* watching…” cried Mr. Jonson. Vincent feeling saddened over the fact *that* he did not pay attention *to the* shift in the conversation, thus leading to him making an old senile man cry, replied with a sense of urgency, “There is nothing for you to worry about, Mr. Jonson. I will leave the station now and be there within the hour.” 
Vincent hung up the phone and swiftly grabbed his coat. He imagined this case would be a lot of calming down Mr. Jonson and dismissing the actual complaint. Grabbing his hat, Vincent walked out of his office, and continued out of the station. Walking out of the building, Vincent noticed that the sky had turned an odd gray with a tint of green. It seemed as though nature itself was dissatisfied with the state of the town. The wind was silent and stale. The streets were empty and the town lay still, unmoving. Feeling suddenly unnerved, Vincent looks at his watch. “Why is it so quiet when it's only 7 o'clock? Hopefully, I can make it home before the graveyard *shift.” His light blue ’69 Chevy Caprice* roared to life and began pushing down the main street of Janesborough. Driving down the street, the roads were usually full of life. And if not lively, they at least had people passing on the other side of the street. Tonight there was nothing, but the light hum of his *Chevy lightly* passing over the dark pavement. 
Vincent pulls up to Mr. Jonson's home and knocks on the door. The knock echoes throughout the house.  The echo continues as if the sound had fallen down a well and reverberated into infinity. Waiting a few more moments, Vincent considers his options. “Hmmm, waiting here isn't getting me anywhere. I'm sure the old fogey has already cried himself to sleep with his ridiculous fears. I might as well check out this house across the street. *It’s a* little *too* early to head back home.”
Vincent continues across the street, still finding it strange that no one is on the streets. Walking to the front door, a debilitating stench struck his nose. “*Dear* God, what in the Hell is that dreadful stench?!” yelled Vincent. Covering his nose he decided to cover his nose and knock on the door. Reaching *to* knock on the door, a strong wind blew from behind Vincent, blowing him forward and *opening the door*. As the door swung open, the stinging smell *blew* full force from inside the home directly into *Vincent’s* face. His eyes began to water and nose began to burn uncontrollably. He suddenly felt nauseous, but continued inside the home because of his need to have an actual case. 
Inside Vincent slides his hand upon the walls looking for a light switch. “Can't see a damn thing in this trifling house…” Vincent complains. Coming across a switch, Vincent flicks the switch back and forth, which to his dismay does nothing. “Damn, damn, damn! Now I have to find a damn breaker-box. It should be on the outside of the house.” 
The smell forcibly repels Vincent from the house like a wolf protecting his territory. The detective continues around the house and searches the outside of the house for a breaker-box. Reaching to slide his hand across the outside of the home as well, *Vincent’s hand* swipes across a sharp piece of pipe traveling on the outskirts of the home. “Fuck!” screams Vincent. “What the hell is up with this home?!”  Slightly bleeding, Vincent finds the breaker-box and opens the covering, revealing the electrical components *inside*.

**Comment by jf-42** [comment]
1969_chevrolet_caprice
'69 Chevy Caprice

**Comment by KyWalker** [comment]
I like the story so far, but it really threw me off when you switched from a first person P.O.V to third person P.O.V. We probably want to keep that consistent throughout the whole story.

File: Section1.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <head>
+        <meta charset="utf-8">
+        <title>Section 1 of Something Inhuman</title>
+    </head>
+    <body>
+
+Murder Mystery Web 2 Section 1
+
+
+The Start of Something Inhuman
```
Comment (dwds) [inline Section1.html]: &lt;h1&gt;

File: Section1.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+
+
+“In my town there is not much to hold many people's attention. The community college, the insignificant downtown sector, the farmlands, and the forest all come together each like a stitch in a Christmas quilt. The town as a whole mostly holds a special place in the townsfolk's hearts. The older generation has lived here since their parents' generation and thereon, so everyone looks out for one another. Though this is good for a starting detective such as myself, it also makes my job just a little bit monotonous.
```
Comment (dwds) [inline Section1.html]: Markup all paragraphs with &lt;p&gt;

File: Section1.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Section 1 of Something Inhuman</title>
+    </head>
+    <body>
+
+Murder Mystery Web 2 Section 1
```
Comment (dwds) [inline Section1.html]: delete

**Comment by vhoge94** [comment]
The first act really pulls me into the story. Like like the add ins jf-41. Good Job!

**Comment by Jax-Tsukuyomu** [comment]
Everything should be fixed and taken care of. Take another look if everyone would.

File: Section1.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Section 1 of Something Inhuman</title>
+    </head>
+    <body>
+<h1 id="title">
+The Start of Something Inhuman
+</h2>
```
Comment (dwds) [inline Section1.html]: Fix closing tag.

**Comment by dwds** [comment]
Congratulations @Jax-Tsukuyomu, your branch has been merged to master!

**Pull request merged**